node_modules/gh-orgunit/lib/api.js

```diff
@@ -557,51 +563,90 @@ var addOrgUnitSeries = module.exports.addOrgUnitSeries = function(ctx, id, serie
             return callback(err);
         }
 
-        // Ensure the series exist
-        SeriesDAO.getSeries(series, function(err, series) {
+        // Ensure the "from" organisational unit exists if it was specified and contains the specified series
```

Comment:
This can use some better wording:

e.g. `If the event series is borrowed from a different organisational until, ensure that the organisational until exists and contains the series`